src/app/etl-api/etl-api.module.ts

```diff
@@ -78,7 +82,9 @@ import { OncologySummaryResourceService } from './oncology-summary-resource.serv
     HivEnhancedReportService,
     HivEnhancedReportService,
     OncologySummaryResourceService,
-    RadiologyImagingResourceService
```

Comment:
why are we removing thise radiology resource service? some code might break somewhere

Reply:
reinstated this line back